app/models/snp.rb

```diff
@@ -43,6 +46,15 @@ def self.update_frequencies
     end
   end
 
+  def self.update_phenotypes
+    max_age = 31.days.ago
+
+    Snp.select([:id, :phenotype_updated]).where(phenotype_updated < max_age)
```

Comment:
Shouldn't this be `.where('phenotype_updated < ?', max_age)`? You also don't need the square brackets, `.select(:id, :phenotype_updated)` works as well.